pkg/reconciler/taskrun/taskrun.go

```diff
@@ -465,67 +498,38 @@ func (c *Reconciler) handlePodCreationError(tr *v1alpha1.TaskRun, err error) {
 	c.Logger.Errorf("Failed to create build pod for task %q: %v", tr.Name, err)
 }
 
-func updateTaskRunResourceResult(taskRun *v1alpha1.TaskRun, podStatus corev1.PodStatus) error {
```

Comment:
This just moved down - no changes.
I consolidated all func (c *Reconciler) together.